orc8r/cloud/helm/orc8r/templates/ctraced.service.yaml

```diff
@@ -0,0 +1,31 @@
+{{/*
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+*/}}
+
+{{- include "orc8rlib.service" (list . "ctraced.service") -}}
+{{- define "ctraced.service" -}}
+metadata:
+  name: orc8r-ctraced
+  labels:
+    {{- with .Values.ctraced.service.labels }}
+{{ toYaml . | indent 4}}
+    {{- end}}
+  {{- with .Values.ctraced.service.annotations }}
+  annotations:
+{{ toYaml . | indent 4}}
+  {{- end }}
+spec:
+  ports:
+    - name: grpc
+      port: 9180
+      targetPort: 9109
```

Comment:
Target port needs to be the same as the service port defined in the service_registry.yaml (9118)